src/form-field-schema.js

```diff
@@ -67,7 +67,7 @@ export default class FormFieldSchema {
         this.addElementColumn(element, 'country');
       }
 
-      if (this.fullSchema && (element.isPhotoElement || element.isVideoElement || element.isAudioElement)) {
+      if (this.fullSchema && (element.isPhotoElement || element.isVideoElement || element.isAudioElement || element.isAttachmentElement)) {
```

Comment:
The next if, in operator.js, has element.isSignatureElement as well.  It seems odd that it's not here.

Reply:
it looks like isSignatureElement is handled separately on line 75? Is that what you mean?

Reply:
Yeah, that works.  I don't understand what this is doing, just looking for inconsistencies.